go.sum

```diff
@@ -1,7 +1,3 @@
-bitbucket.org/creachadair/stringset v0.0.8 h1:gQqe4vs8XWgMyijfyKE6K8o4TcyGGrRXe0JvHgx5H+M=
```

Comment:
why go sum is deleted?

Reply:
It should be in sync with `go.mod`. I removed it from there and `go mod tidy` removed it from here.